documentation/api/metrics/v1/index.markdown

```diff
@@ -9,6 +9,7 @@ canonical: "/puppetdb/latest/api/metrics/v1/index.html"
 Querying PuppetDB metrics is accomplished by making an HTTP request
 to paths under the `/metrics/v1` REST endpoint.
 
+
```

Comment:
Wonder if that's due to the rebase onto AJs work.  In any case, I'll fix it shortly.